Google.Api.Gax.Grpc/Rest/RestChannel.cs

```diff
@@ -124,6 +132,41 @@ private async Task<ReadHttpResponseMessage> SendAsync<TRequest>(RestMethod restM
             }
         }
 
+        private async Task<HttpResponseMessage> SendAsync<TRequest>(RestMethod restMethod, string host, CallOptions options, TRequest request,
```

Comment:
Could you move this to before AsyncServerStreamingCall to make the change easier to understand? We can then have a purely-refactoring PR that moves it.